dev/README.md

```diff
@@ -0,0 +1,14 @@
+# Build environnement for lizmap
+
+## Build Lizmap package
+
+from the `dev` directory
+```
+make build
+```
+
+or from the parent directory:
+```
+make -C dev build
+```
+
```

Comment:
```suggestion
A `build` directory is created with zip packages inside.

```

Just a quick note to explain what the user should check after running this command.
```